pkg/apis/flowcontrol/v1alpha1/defaults.go

```diff
@@ -18,21 +18,30 @@ package v1alpha1
 
 import (
 	"k8s.io/api/flowcontrol/v1alpha1"
-	"k8s.io/kubernetes/pkg/apis/flowcontrol"
+)
+
+// Default settings for flow-schema
+const (
+	FlowSchemaDefaultMatchingPrecedence int32 = 1000
+)
+
```

Comment:
Would it be better to put these constants in `k8s.io/api/flowcontrol/v1alpha1` along with the definitions of the v1alpha1 object types?